src/server/server.ts

```diff
@@ -73,6 +73,9 @@ const ndlaMiddleware = [
       allowedBodyContentTypes.includes(req.headers['content-type'] ?? ''),
   }),
   helmet({
+    crossOriginEmbedderPolicy: {
+      policy: 'credentialless',
+    },
```

Comment:
Dette funker lokalt for meg.
```suggestion
    crossOriginEmbedderPolicy: false,
```